regression/pages/studio/asset_index_studio.py

```diff
@@ -53,23 +52,17 @@ def get_files_count(self):
         """
         return len(self.q(css='#asset-table-body tr'))
 
-    def delete_file(self, file_indexes):
+    def delete_file(self):
         """
-        Delete the file(s)
-        Arguments:
-            file_indexes (list): list of indexes of file(s)
-                we want to delete.
-        """
-        more_then_one_file = False
-        for index in file_indexes:
-            if more_then_one_file and index != 0:
-                index = index - 1
-            click_css_with_animation_enabled(
-                self, '.remove-asset-button.action-button', index, False)
-            confirm_prompt_with_animation_enabled(
-                self, require_notification=False)
+        Deletes the file till delete asset button is found on page
```

Comment:
This is another instance of: sensible programming, bad test pattern. The tests should be built in such a way that we know how many files should be appearing, and we know how many times we should be clicking. Otherwise, we may hide bugs on this action. I should have noticed this on the original implementation as well. 

It could be a "less smart" function: click the delete button. Then in the test, you do a wait to see if the file has been deleted. If you have several files, you would cycle through that on the test, rather than the helper function. And you'd do it a known amount of times. I hope this feedback makes sense?